src/codeu/chat/util/Tokenizer.java

```diff
@@ -0,0 +1,79 @@
+//there are two parts in the next() funtion that I still need to complete
+//I commented in the commented section about this
+
+package codeu.chat.util;
+
+import java.io.IOException;
+import java.util.logging.Level;
+
+public final class Tokenizer {
+  
+  private StringBuilder token;
+  private String source;
+  private int at;
+
+  
+  
+  public Tokenizer(String s) { 
+    source = s;
+  }
+  
+  private int remaining() {
+      return source.length() - at;
+    }
```

Comment:
These two lines need their indentation fixed, also there should be an empty line after this function definition